src/lib/source-handlers/github/get-repo-metadata.ts

```diff
@@ -5,7 +5,7 @@ import type { RepoMetaData, Target } from '../../types';
 import { getGithubToken } from './get-github-token';
 import { getGithubBaseUrl } from './github-base-url';
 
-const githubClient = Octokit.plugin(retry);
+const githubClient = Octokit.plugin(retry as any);
```

Comment:
Is this how they recommend to set the plug in? With  `as any`? It concerns me a bit that the types don't match 🤔

Reply:
The type errors appeared after upgrading the TS version, nothing changed functionally with this package or the retry plugin but I had to add this typecast to shut TS up and be able to move on with the upgrade.